_posts/pl/2021-12-XX-transactions.md

```diff
@@ -0,0 +1,217 @@
+---
```

Comment:
Zastanawiam się czy tego artykułu nie warto by było podzielić na dwie lub nawet trzy części. IMO za dużo informacji do przetrawienia

Reply:
chyba masz rację, może podzielić to jak poniżej?
1. Ogólne info + transakcje w JDBC 
2. Transakcje w Springu